swagger.yaml

```diff
@@ -959,6 +992,53 @@ definitions:
             type: string
             description: 9 digits SSN
             example: '123456789'
+  PhonesResultObject:
+    properties:
+      data:
+        type: array
+        items:
+          properties:
+            id:
+              type: string
+              description: Globally unique Id for the phone numbers
+            type:
+              type: string
+              example: phone
+            attributes:
+              properties:
+                areaCode:
+                  type: string
+                  example: 541
+                phoneNumber:
+                  type: string
+                  example: 4443333
+                fullPhoneNumber:
+                  type: string
+                  description: Combination of area code and phone number
+                  example: 5414443333
+                phoneExtension:
+                  type: string
+                  example: 20
+                primaryIndicator:
+                  type: string
+                  example: Y
+                phoneType:
+                  type: string
+                  example: CM
+                  enum: [PA, RN, RB, MG, RE, RK, VO, VP, VF, CM, EO, RH, EF, MP]
+                phoneTypeDescription:
+                  type: string
+                  example: Current
+                addressType:
+                  type: string
+                  example: CM
+                  enum: [RN, MG, VO, RE, RB, VP, VF, RK, CM, EF, RH, EO, PA, MP]
+                addressTypeDescription:
+                  type: string
+                  example: Current Mailing
+                activityDate:
```

Comment:
Can you rename this as `lastModified` to make it consistent with address endpoint?